starport/services/serve/dev.go

```diff
@@ -1,57 +1,97 @@
 package starportserve
 
 import (
-	"encoding/json"
 	"net/http"
 
 	"github.com/gobuffalo/packr/v2"
 	"github.com/gorilla/mux"
+	"github.com/tendermint/starport/starport/pkg/httpstatuschecker"
+	"github.com/tendermint/starport/starport/pkg/xhttp"
 )
 
-// newDevHandler creates a new development server handler.
-func newDevHandler(app App) http.Handler {
+const (
+	appNodeInfoEndpoint = "/node_info"
+)
+
+// serviceStatus keeps the state of development environment and services needed
+// for development.
+type serviceStatus struct {
+	Env                    env  `json:"env"`
```

Comment:
Can we move `env` outside of `status` so that the returned object looks like this:

```js
{
  "env": {
    "chain_id": "blog",
    // ...
  },
  "status": {
    "is_consensus_engine_alive": true,
    // ...
  }
}
```

Reply:
solved by 6134ae5